app/views/layouts/_header.html.erb

```diff
@@ -177,6 +177,7 @@
                     <a class="btn btn-default" href="/signup" tabindex="4"><%= t('layout._header.login.sign_up') %></a>
                   </p>
 
+                  <p><a href="/auth/facebook">Login with Facebook</a></p>
```

Comment:
Here -- we could do `<% if params[:fb] %> ... <% end %>` to only show this when that flag is set, you know?

Reply:
Yeah!!!   I also want to test this feature on production before launching to public. Thanks for the hint. I will try to do it.